test/torch/tensors/test_add.py

```diff
@@ -0,0 +1,62 @@
+import random
+
+import pytest
+import torch
+import torch.nn.functional as F
+import syft
+
+from syft.frameworks.torch.tensors import AdditiveSharingTensor
+
+
+class TestAdditiveSharingTensor(object):
```

Comment:
Alexis put a huge amount of work refactoring all of our tests to use the pytest style. This file should be reformatted for consistency